fe/fe-core/src/main/java/org/apache/doris/catalog/Env.java

```diff
@@ -290,10 +291,13 @@ public class Env {
     public static final String CLIENT_NODE_HOST_KEY = "CLIENT_NODE_HOST";
     public static final String CLIENT_NODE_PORT_KEY = "CLIENT_NODE_PORT";
 
+    private static final String VERSION_DIR = "/VERSION";
+    public static String latestFeVersion = "2_0_0";
```

Comment:
do not add this variable, should generate the version from global variables. For example, show variable will show version info.
You add this variable and then when we publish a version, we has to remember to modify this variable. It is very difficult to maintain it.